### PR TITLE
Send more detailed OS information to monitor

### PR DESCRIPTION
**Pull request by Wiezzel**
TODO: Check if this works on macOS, add necessary fixes & test.

### CODE REVIEW

**Comment by Wiezzel** [comment]
See https://github.com/golemfactory/golem-monitor/pull/60 for respective monitor changes.

**Comment by Wiezzel** [comment]
@maaktweluit Could you pick up the "TODO"?

**Comment by maaktweluit** [comment]
@Wiezzel yes, will have a look!

**Comment by maaktweluit** [comment]
I have logged the os_info after it is serialised and see enough info to get the version:

```INFO     [golem.monitor.model.nodemetadatamodel] OS_INFO '{"type": "OSInfo", "obj": {"platform": "darwin", "system": "Darwin", "release": "18.0.0", "version": "Darwin Kernel Version 18.0.0: Wed Aug 22 20:13:40 PDT 2018; root:xnu-4903.201.2~1/RELEASE_X86_64", "windows_edition": null, "linux_distribution": null}}'```

`18.0.0 == mojave`

Source: https://en.wikipedia.org/wiki/Darwin_(operating_system)

File: golem/tools/os_info.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from golem.core.common import is_windows, is_linux
+
+
+class OSInfo:
+
+    WIN_VERSION_KEY = r'SOFTWARE\Microsoft\Windows NT\CurrentVersion'
+    WIN_EDITION_ID = 'EditionID'
+
+    # pylint: disable=too-many-arguments,redefined-outer-name
```
Comment (maaktweluit) [inline golem/tools/os_info.py]: why do you need `redefine-outer-name`?

**Review by maaktweluit — APPROVED** [review]
LGTM!

Just a small linter question

File: golem/tools/os_info.py (hunk 1 above)
Comment (Wiezzel) [reply golem/tools/os_info.py]: Because of `platform`.

Comment (maaktweluit) [reply golem/tools/os_info.py]: maybe `_platform` would be a better solve then?

Comment (Wiezzel) [reply golem/tools/os_info.py]: Maybe, I'm not sure. `¯\_(ツ)_/¯`

Comment (jiivan) [reply golem/tools/os_info.py]: I second to that this lint is valid, and would like to avoid that redefinition.

**Review by jiivan — APPROVED** [review]

File: golem/tools/os_info.py (hunk 1 above)
Comment (Wiezzel) [reply golem/tools/os_info.py]: :ballot_box_with_check: Fixed

Comment (maaktweluit) [reply golem/tools/os_info.py]: i like the solve a lot more then my proposal! 👍

**Pull request merged**